ts/src/binance.ts

```diff
@@ -8511,6 +8580,7 @@ export default class binance extends Exchange {
                 const firstElement = response[0];
                 const errorCode = this.safeString (firstElement, 'code');
                 if (errorCode !== undefined) {
+                    this.throwExceptionByUrl (url, 'exact', errorCode, this.id + ' ' + body);
```

Comment:
@ttodua Now I'm reconsidering the need for this method, what if we used the `getExceptionsByUrl` method inside `handleErrors` to get the correspondent exceptions and we keep the original `throwExactlyMatchedException` method and we just change the argument provided to it. 

Example:

```Typescript
        const exactExceptions = this.getExceptionsByUrl(url, 'exact');
        const broadExceptions = this.getExceptionsByUrl(url, 'broad');
        this.throwBroadlyMatchedException (exactExceptions, body, feedback);
        this.throwExactlyMatchedException (broadExceptions, errorCode, feedback);
```

Reply:
ofc, I though i could do the `throw` inside that method too, but i think thats better what you say, so the new method will be just to get the exception tree for market-type

Reply:
pushed now, please lmk if that's good now